controllers/provider-aws/pkg/infrastructure/add.go

```diff
@@ -41,7 +41,7 @@ var (
 
 func init() {
 	addToManagerBuilder.Register(Add)
-	terraformerImage, err := imagevector.ImageVector.FindImage(aws.TerraformerImageName, "", "")
+	terraformerImage, err := imagevector.GetImageVector().FindImage(aws.TerraformerImageName, "", "")
```

Comment:
Write a method for this as in the other controllers (i.e. `TerraformerImage()`).

Reply:
Again, this is not part of my change, I only changed it because unit tests were failing otherwise. Perhaps it makes sense, but I think it doesn't belong in this PR.

Reply:
Again, you renamed it from `ImageVector` to `GetImageVector` - This has to be changed by you.

Reply:
I don't think so - this is only an adaptation to make any tests work.